TrackingTools/TrackFitters/plugins/KFFittingSmootherESProducer.cc

```diff
@@ -65,7 +65,7 @@ namespace {
       desc.add<int>("MaxNumberOfOutliers", 3);
       desc.add<int>("MinDof", 2);
       desc.add<bool>("NoOutliersBeginEnd", false);
-      desc.add<int>("MinNumberOfHits", 5);
+      desc.add<int>("MinNumberOfHits", 3);
```

Comment:
it looks like this is cloned all over the place, like `GsfElectronFittingSmoother` or `KFFittingSmootheForSTA`, which all could trigger an increase in fakes (unless the number of hits 5->3 is shadowed by a tighter cut somewhere else).

It would be better to change this only in the iter tracking context.

Reply:
So, a bugfix is quite likely needed now.

Reply:
> So, a bugfix is quite likely needed now.

Yes!
Please @slava77 @mmasciov provide that fix
Please @rappoccio wait for that fix before proceeding with building the pre-release!

Reply:
@slava77, @perrotta, @rappoccio: PR #39611 now only applies this change to `RKFittingSmoother`, without affecting `GsfElectronFittingSmoother` or `KFFittingSmootheForSTA`.